### DISCUSSION TITLE
Create 64x64 brain MRI scans that emulate low-field MRI scans

### DISCUSSION THREAD

**Discussion by arp95** [facebookresearch/fastMRI#286] [General]
Hi,

I was trying to follow the fastMRI_tutorial.ipynb where we obtain the kspace of the MRI scan, inverse fourier transform, obtain complex image and then do root mean square from all coils to obtain the scan. I was trying to build a neural network where the low-field images are input and the output is high-field image. For this i obtain 64x64 center region from the mri scans and 128x128 center regions as the corresponding high resolution image.
For low resolution scan, what is the noise i should add to emulate different types of MRI scans? Please let me know.

**Comment by mmuckley** [comment]
Hello @arp95, I'm a little confused on the use of the term "low-field". Do you mean low-resolution? In the MRI context low-field has a very different meaning from low-resolution.

For simulating low-resolution data you can use the procedure described in [Bakker et al.](https://proceedings.mlr.press/v172/bakker22a) in Appendix C.1. The noise in MRI is Gaussian, with some correlations between the coils. This means that you'll get the same distribution you would expect with each FFT operation.

In practice with aspects like frequency oversampling and such you'll get a slightly different distribution, but the cropping procedure in Bakker et al. is a reasonable starting point for working with different resolutions.

**Comment by arp95** [comment]
Hi @mmuckley . Thanks for your help.
So according to my task, I am generating center crops 64x64 of the multi-coil brain MRI scans that correspond to low resolution images for my task and same 128x128 center crops that correspond to high resolution images. I will use these pair for training a neural network.
For 64x64 low resolution images, can I generate undersampled k-space data that was done by the paper you referred to? So that the model generalises to different images? This is the paper I am referring to https://www.nature.com/articles/s41598-022-10298-6. This uses fastMRI dataset, generated 64x64 and 128x128 and adds noise to 64x64 images. The noise can be the "RandomMaskFunc(center_fractions=[0.04], accelerations=[8])" as mentioned in fastMRI tutorial or am i wrong?
